blog/_posts/2017-08-14-with-love-team-bundledore-india.md

```diff
@@ -0,0 +1,77 @@
+title: With love, Team Bundledore, India !
```

Comment:
Heyy, I have a slight doubt on this one. Is this okay ?

`---
title: With love, Team Bundledore, India
layout: post
created_at: Tuesday August 15 2017
permalink: blog/2017-08-09-with-love-team-bundldore-india
current: blog
author: Bundledore
twitter: Team Bundledore
categories:
- student-posts
--- `